build/media-suite_compile.sh

```diff
@@ -303,7 +303,7 @@ _check=(libgnutls.{,l}a gnutls.pc)
 if enabled_any gnutls librtmp || [[ $rtmpdump = y ]] || [[ $curl = gnutls ]] &&
     ! files_exist "${_check[@]}" &&
     do_wget -h 881b26409ecd8ea4c514fd3fbdb6fae5fab422ca7b71116260e263940a4bbbad \
-    "https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/gnutls-3.6.7.1.tar.xz"; then
+    "https://www.gnupg.org/ftp/gcrypt/gnutls/v3.6/gnutls-3.6.7.1.tar.xz" gnutls-3.6.7.tar.xz; then
```

Comment:
Why?

Reply:
Because it will unpack as 3.6.7 while do_wget will try to changedir to 3.6.7.1 which doesn't work. It's all tested, I'm done.

Please check https://gitlab.com/gnutls/gnutls/issues/745

Reply:
Oh ok. Yeah, that's exactly why I supported that usecase with the second argument.

Reply:
:+1:  Can (hopefully) be removed in gnutls 3.6.8 again ;-)